src/main/java/upm/softwaredesign/finalproject/service/ActorService.java

```diff
@@ -0,0 +1,52 @@
+package upm.softwaredesign.finalproject.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import upm.softwaredesign.finalproject.model.ActorEntity;
+import upm.softwaredesign.finalproject.repository.ActorRepository;
+
+@Service
+public class ActorService{
+
+	private final ActorRepository repository;
+	
+	@Autowired
+	public ActorService(ActorRepository repository) {
+		this.repository = repository;
+	}
+
+
+	public Optional<ActorEntity> findById(Integer id ) {
+		return this.repository.findById(id);
+	}
+
+    public void save(ActorEntity actor) {
```

Comment:
Okay here is again the problem of the Actor and ActorEntity. So your module receives an Actor object. Actor object or any other object in the system should not be aware or care at all about the database. If inside the model folder you are implementing classes that are representing mappings between the database and your object's that fine, but I think that is wrong that you change Piotr's module classes or someone else's to implement those mappings. So you should receive an Actor object, maybe wrap it into your ActorEntity object and then save it to the database. That's my understanding, we can further discuss with others.